infrared.cfg.example

```diff
@@ -2,7 +2,7 @@
 # ===========================
 
 [defaults]
-settings  = settings
+settings  = settings:private_settings
```

Comment:
cfg.example should be ready to use. private_settings isn't ready to use... 
Please remove it.